src/server/page.ts

```diff
@@ -203,6 +203,11 @@ export class Page extends SdkObject {
       this.emit(Page.Events.Close);
   }
 
+  initializedOrNull() {
```

Comment:
I meant that we should _only_ report the page after it is initialized. To do that:
- Introduce `private _initialized = false;` in the `Page` class.
- At the line 198, right before emitting `BrowserContext.Events.Page`, assign `_initialized` to `true`.
- `return this._initialized ? this : undefined` here.
- I also just realized that we should send `undefined` instead of `null` in this case, so let's call the method `initializedOrUndefined`.